qprofiler2/src/org/qcmg/qprofiler2/summarise/ReadGroupSummary.java

```diff
@@ -1,187 +1,97 @@
 package org.qcmg.qprofiler2.summarise;
 
-import java.util.concurrent.atomic.AtomicInteger;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.atomic.AtomicLong;
 import org.qcmg.common.model.QCMGAtomicLongArray;
-import org.qcmg.common.util.QprofilerXmlUtils;
-import org.qcmg.picard.util.PairedRecordUtils;
+import org.qcmg.common.util.Pair;
+import org.qcmg.qprofiler2.util.SummaryReportUtils;
 import org.qcmg.qprofiler2.util.XmlUtils;
 import org.w3c.dom.Element;
 import htsjdk.samtools.CigarElement;
 import htsjdk.samtools.CigarOperator;
 import htsjdk.samtools.SAMRecord;
 
 public class ReadGroupSummary {
+
+	public final static int errReadLimit  = 10;	
 	//xml node name 	
-	public final static String node_readgroup = "readGroup";
-	public final static String node_f5f3 = "f5f3Pair";
-	public final static String node_f3f5 = "f3f5Pair";
-	public final static String node_inward = "inwardPair";
-	public final static String node_outward = "outwardPair";		
+	public final static String node_readgroup = "readGroup";		
```

Comment:
camelCase for all variables pls - constants should be UPPER_CASE

Reply:
done